src/components/Chat/RoomMenu.tsx

```diff
@@ -118,9 +123,12 @@ export const RoomButtons: React.FC<RoomMenuProps> = ({ onChange, onClose }) => {
       ref={ref}
     >
       <div className="room_menu__header">
-        <button className="room_menu__toggle" onClick={onClose}>
-          <BsArrowBarRight />
-        </button>
+        {!mainActiveTopic && (
+          <button className="room_menu__toggle--on" onClick={handleToggleClick}>
```

Comment:
modifiers are used to define styles for alternate states ie `disabled`... in most cases defining them statically makes no sense